regression/tests/main/tensorflow/tf2-examples/densenet/regression.yaml

```diff
@@ -29,13 +29,13 @@ components:
                 - wandb-init-callback.patch
 variants:
     - base:
-        - python3s
+        - python3ns
         - wandb-run
         - python-path
         - tensorflows
         - tfds
     - callback:
-        - python3s
+        - python3ns
```

Comment:
there is a problem with earlier version of tensorflow and latest tensorflow-datasets since it installs a later version of protobufs... we should split the tests?